lib/tpm2_convert.h

```diff
@@ -25,6 +25,25 @@ enum tpm2_convert_sig_fmt {
     signature_format_err
 };
 
+typedef enum tpm2_convert_pcrs_output_fmt tpm2_convert_pcrs_output_fmt;
+enum tpm2_convert_pcrs_output_fmt {
+    pcrs_output_format_values,
+    pcrs_output_format_serialized,
```

Comment:
I wish we could find a better naming scheme than values and serialized, but since I don't have a better suggestion and the binary formats are obtuse anyways, it lives on.